server/middleware/setupPdfRenderer.ts

```diff
@@ -1,33 +1,49 @@
 import { Request, Response, NextFunction } from 'express'
-import GotenbergClient, { PdfOptions } from '../data/gotenbergClient'
+import GotenbergClient from '../data/gotenbergClient'
 import logger from '../../logger'
 
+export type PdfOptions = {
+  headerHtml?: string
+  footerHtml?: string
+  marginTop: number
+  marginBottom: number
+  marginLeft: number
+  marginRight: number
+  filename: string
```

Comment:
The filename and content disposition (whether its is a download (attachment) or to be displayed in a tab (inline)) are now able to be specified by the calling code 👍